ai_genomics/analysis/gtr_definitions.py

```diff
@@ -0,0 +1,130 @@
+# Script to explore AI definition
+
+import json
+import logging
+import random
+import pandas as pd
+import os
+from collections import Counter
+
+from ai_genomics.pipeline.gtr import fetch_gtr
+from ai_genomics import config, PROJECT_DIR
+
+os.makedirs(f"{PROJECT_DIR}/inputs/data/gtr", exist_ok=True)
```

Comment:
`PROJECT_DIR` is a path object which has mkdir functionality built in. It can be written like this:
```suggestion
GTR_INPUTS_DIR = PROJECT_DIR / "inputs/data/gtr"
GTR_INPUTS_DIR.mkdir(exist_ok=True)
```

Also I've suggested creating `GTR_INPUTS_DIR` as the same path is used later to save a file.